### DISCUSSION TITLE
AppShell padding x and y

### DISCUSSION THREAD

**Discussion by kimfucious** [mantinedev/mantine#2261] [Feature requests]
I would be nice if padding was not overall, and could be controlled both x and y.

At present, it seems that the only prop is `padding`, which is overall.

**Comment by rtivital** [comment]
You can set padding={0} on the AppShell and then wrap your content with Box, this way you can add any kind of padding and other styles

**Reply by kimfucious** [reply]
Yep, thanks.  

That's what I'm doing.

Just thought I'd make a suggestion.

I'm digging Mantine.  

Thanks for making it.

Keep up the great work!
